### PR TITLE
[Doc] Add apache security link

### PR DESCRIPTION
**Pull request by EricJoy2048**

### CODE REVIEW

**Review by TaoZex — APPROVED** [review]
LGTM

**Review by TyrantLucifer — CHANGES_REQUESTED** [review]
revert `package-lock.json`

File: package-lock.json [hunk 1]
```diff
@@ -9433,4 +9433,4 @@
       "integrity": "sha1-0R1zgf/tFrdC9q97PyI9XNn+mSA="
     }
   }
-}
+}
```
Comment (TyrantLucifer) [inline package-lock.json:9436]: revert

Comment (EricJoy2048) [reply package-lock.json]: done

**Comment by TyrantLucifer** [comment]
Waiting CI

**Review by TyrantLucifer — APPROVED** [review]

**Pull request merged**